forui/lib/src/widgets/pagination/pagination_style.dart

```diff
@@ -29,12 +29,21 @@ final class FPaginationStyle with Diagnosticable, _$FPaginationStyleFunctions {
   @override
   final BoxConstraints contentConstraints;
 
+  /// The tappable's action style.
+  @override
+  final FTappableStyle actionTappableStyle;
+
+  /// The tappable's page style.
+  @override
+  final FTappableStyle pageTappableStyle;
```

Comment:
How is pageTappableStyle different from actionTappableStyle?

Reply:
One represents the `FTappable` for the actual page, and the other for the action buttons.